src/component/feature/FeatureToggleList/FeatureToggleList.tsx

```diff
@@ -1,26 +1,58 @@
-import { useContext } from 'react';
-import PropTypes from 'prop-types';
+import { Dispatch, SetStateAction, useContext, VFC } from 'react';
 import classnames from 'classnames';
 import { Link } from 'react-router-dom';
 import { List, ListItem } from '@material-ui/core';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
-import FeatureToggleListItem from './FeatureToggleListItem';
+import { IFeatureToggle } from 'interfaces/featureToggle';
+import { IFlags } from 'interfaces/uiConfig';
 import { SearchField } from 'component/common/SearchField/SearchField';
-import FeatureToggleListActions from './FeatureToggleListActions';
-import ConditionallyRender from 'component/common/ConditionallyRender/ConditionallyRender';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
 import PageContent from 'component/common/PageContent/PageContent';
-import HeaderTitle from 'component/common/HeaderTitle';
-import loadingFeatures from './loadingFeatures';
+import { HeaderTitle } from 'component/common/HeaderTitle/HeaderTitle';
 import AccessContext from 'contexts/AccessContext';
-import { useStyles } from './styles';
 import ListPlaceholder from 'component/common/ListPlaceholder/ListPlaceholder';
+import { IFeaturesFilter } from 'hooks/useFeaturesFilter';
+import { FeatureToggleListItem } from './FeatureToggleListItem/FeatureToggleListItem';
+import { FeatureToggleListActions } from './FeatureToggleListActions/FeatureToggleListActions';
 import { CreateFeatureButton } from '../CreateFeatureButton/CreateFeatureButton';
 import { useCreateFeaturePath } from '../CreateFeatureButton/useCreateFeaturePath';
+import { IFeaturesSort } from 'hooks/useFeaturesSort';
+import { useStyles } from './styles';
+
+interface IFeatureToggleListProps {
+    features: IFeatureToggle[];
+    loading?: boolean;
+    flags?: IFlags;
+    filter: IFeaturesFilter;
+    setFilter: Dispatch<SetStateAction<IFeaturesFilter>>;
+    sort: IFeaturesSort;
+    setSort: Dispatch<SetStateAction<IFeaturesSort>>;
+    onRevive?: (feature: string) => void;
+    isArchive?: boolean;
+}
 
-const FeatureToggleList = ({
+const loadingFeaturesPlaceholder: IFeatureToggle[] = Array(10)
+    .fill({
+        createdAt: '2021-03-19T09:16:21.329Z',
+        description: ' ',
+        enabled: true,
+        lastSeenAt: '2021-03-24T10:46:38.036Z',
+        name: '',
+        project: 'default',
+        stale: true,
+        strategies: [],
+        variants: [],
+        type: 'release',
+        archived: false,
+        environments: [],
+        impressionData: false,
+    })
+    .map((feature, index) => ({ ...feature, name: `${index}` })); // ID for React key
```

Comment:
I don't think objectId will work for this, because it's a map of identical things - for loader. But thx for pointing this out. I didn't know about this util :)